torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1348,6 +1360,22 @@ class IrParser {
           },
           nullptr,
           nullptr);
+
+    }
+
+    {
```

Comment:
nitpick, can we merge all three ops together since they have identical parsing rule. :pray: